<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AaServerWindowApp</class>
 <widget class="QMainWindow" name="AaServerWindowApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>6</number>
      </property>
      <widget class="QTextEdit" name="m_txtOutput"/>
      <widget class="QTextEdit" name="m_txtInput">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>449</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSend_File"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSend_File"/>
  </widget>
  <action name="actionSend_File">
   <property name="icon">
    <iconset resource="../../icons/baseIcon.qrc">
     <normaloff>:/openFile</normaloff>:/openFile</iconset>
   </property>
   <property name="text">
    <string>Send File</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../icons/baseIcon.qrc"/>
 </resources>
 <connections/>
</ui>
